rhiever/tpot
# Evolve directly on sklearn pipelines

## What does this PR do?

Redesigns how TPOT evaluates pipelines, by changing the DEAP compile function to use the TPOT export utils' generate_pipeline_code function.

The changes also seem to fix the issue the development branch has had with scoring pipelines.

## Where should the reviewer start?

Notable changes/additions:

* All `_call` and `__call__` code removed from the operators (`CombineDFs` operator is now a shell of its former self)
* `_evaluate_individual`
* `_compile_to_sklearn`

## How should this PR be tested?

Tests are not yet updated for these changes, but it would be a good idea to remove the "catch-all" exception handler from `_evaluate_individual` and see what mistakes are happening under the hood.
